recipes/newrelic/infrastructure/alerts/golden.yml

```diff
@@ -78,6 +78,11 @@ install:
             "query": "mutation {alertsPolicyCreate(accountId: {{.NEW_RELIC_ACCOUNT_ID}}, policy: { name: POLICY_NAME incidentPreference: PER_CONDITION_AND_TARGET }) { id name } }"
           }
           EOT
+          if [ $? -eq 0 ]; then 
+            echo "File '/tmp/policy.json' is created"
+          else 
+            echo "Unable to create the file '/tmp/policy.json'. Please retry using the right permissions"
+          fi 
           sudo sed -i "s/POLICY_NAME/\"\"{{.ALERT_POLICY_NAME}}\"\"/g" /tmp/policy.json
```

Comment:
What happens with these two `sed` commands if we cannot create `/tmp/policy.json`?  I see we eventually exit if a curl fails a few lines down, but am not sure if this error handling handles both sed failures and the curl failure (if that makes sense).  Can we simply exit from the `else` if we're unable to create a temp file?

Reply:
Good call. We can exit from it. Thinking of a better way instead of echoing "unable to create a file".